nemo_curator/utils/distributed_utils.py

```diff
@@ -843,6 +843,7 @@ def write_to_disk(
     write_to_filename: Union[bool, str] = False,
     keep_filename_column: bool = False,
     output_type: str = "jsonl",
+    partition_on: Optional[str] = None,
```

Comment:
Add a description for this to docstring.

Reply:
Added it here: https://github.com/NVIDIA/NeMo-Curator/pull/519/commits/d16047343f7413ee3f82df4fc4d38b032419e146